src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/common/ReplyEntity.java

```diff
@@ -0,0 +1,15 @@
+package com.wikia.webdriver.elements.mercury.components.discussions.common;
+
+import lombok.Builder;
+
+public class ReplyEntity {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1118)